site/content/docs/docker/about.md

```diff
@@ -77,6 +77,7 @@ You can run the Automation Framework in docker using the zap.yaml file in the cu
 ```bash
 docker run -v $(pwd):/zap/wrk/:rw -t owasp/zap2docker-stable zap.sh -cmd -autorun /zap/wrk/zap.yaml
 ```
+NOTE: Linux users can use $(pwd).Windows users can replace ($pwd) with 'C:\your\working\directory'.
```

Comment:
This is already mentioned some lines down:
```
Note that `$(pwd)` is only supported on Linux and MacOS - on Windows you will need to replace this with the full current working directory.
```

Reply:
Great! 
Can we Move the line upside  i.e right after the first command?
That would be easier for anyone to understand.